modules/ROOT/examples/csharp/Program.cs

```diff
@@ -104,6 +104,32 @@ private static void GettingStarted()
             // end::getting-started[]
         }
 
+        private static void TestReplicatorConflictResolver()
+        {
+            // tag::replication-conflict-resolver[]
+            var target = new URLEndpoint(new Uri("ws://localhost:4984/mydatabase"));
+            var replConfig = new ReplicatorConfiguration(database, target);
+            replConfig.ConflictResolver = new LocalWinConflictResolver();
+
+            var replicator = new Replicator(replConfig);
+            replicator.Start();
+            // end::replication-conflict-resolver[]
+        }
+
+        private static void TestSaveWithConflictHandler()
+        {
+            // tag::update-document-with-conflict-handler[]
+            using (var document = database.GetDocument("xyz"))
+            using (var mutableDocument = document.ToMutable()) {
+                mutableDocument.SetString("name", "apples");
+                database.Save(mutableDocument, (updated, current) =>
+                {
+                    return true;
+                });
```

Comment:
That's an interesting variant, the [docs on staging](https://docs-staging.couchbase.com/couchbase-lite/2.6/swift.html#conflict-handler) break it down in two code snippets ("conflict handler" and "save method") but it might be overcomplicating things.

The callback syntax is shorter. 1 code snippet.

@rajagp Would you prefer to document it as a callback syntax, as this PR, or into two distinct code snippets, as on staging docs?